Xamarin.Forms.Xaml/MarkupExtensions/FontImageExtension.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+namespace Xamarin.Forms.Xaml
+{
+	[AcceptEmptyServiceProvider]
```

Comment:
❤️

Reply:
I'd add the `Glyph` as `ContentProperty` attribute
```suggestion
	[AcceptEmptyServiceProvider]
	[ContentProperty("Glyph")]
```

Reply:
This is not my first MarkupExtension and I noticed the other platform ones have `[ContentProperty]` but I am not sure exactly what it is for. How is it used?

Adding this change is not a problem at all